handler.py

```diff
@@ -16,6 +16,7 @@
 
 # Constants
 DEFAULT_TTL = 60 * 60 * 24
+SERVICES = ["sendtab"]
```

Comment:
We might want to load this from the environment so we can add supported services without re-uploading a new lambda func.